Src/ReleaseHistory.md

```diff
@@ -16,6 +16,11 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## *v1.9.1*
+
+- FEATURE: Enable response file parsing provided by driver framework. Arguments (e.g., '@Commands.rsp') prefixed with a '@' character will be evaluated as a file path to a text file that contains commands to be injected on the command-line. 
```

Comment:
>Enable [](http://example.com/codeflow?start=11&length=6)

Q: Should we update our command-line docs with this useful new functionality? A: Yes.